{
  "errors": [
    {
      "message": "Cannot set a field which is beyond scope of the data being created or updated",
      "hint": "Use a field which is part of a model being created or updated within this action's inputs",
      "code": "AttributeArgumentError",
      "pos": {
        "filename": "testdata/validation_set_attribute_lhs_not_within_write_scope/schema.keel",
        "offset": 209,
        "line": 11,
        "column": 18
      },
      "endPos": {
        "filename": "testdata/validation_set_attribute_lhs_not_within_write_scope/schema.keel",
        "offset": 228,
        "line": 11,
        "column": 37
      }
    },
    {
      "message": "Cannot set a field which is beyond scope of the data being created or updated",
      "hint": "Use a field which is part of a model being created or updated within this action's inputs",
      "code": "AttributeArgumentError",
      "pos": {
        "filename": "testdata/validation_set_attribute_lhs_not_within_write_scope/schema.keel",
        "offset": 313,
        "line": 14,
        "column": 18
      },
      "endPos": {
        "filename": "testdata/validation_set_attribute_lhs_not_within_write_scope/schema.keel",
        "offset": 333,
        "line": 14,
        "column": 38
      }
    },
    {
      "message": "Cannot set a field which is beyond scope of the data being created or updated",
      "hint": "Use a field which is part of a model being created or updated within this action's inputs",
      "code": "AttributeArgumentError",
      "pos": {
        "filename": "testdata/validation_set_attribute_lhs_not_within_write_scope/schema.keel",
        "offset": 541,
        "line": 21,
        "column": 18
      },
      "endPos": {
        "filename": "testdata/validation_set_attribute_lhs_not_within_write_scope/schema.keel",
        "offset": 568,
        "line": 21,
        "column": 45
      }
    },
    {
      "message": "Cannot set a field which is beyond scope of the data being created or updated",
      "hint": "Use a field which is part of a model being created or updated within this action's inputs",
      "code": "AttributeArgumentError",
      "pos": {
        "filename": "testdata/validation_set_attribute_lhs_not_within_write_scope/schema.keel",
        "offset": 744,
        "line": 27,
        "column": 18
      },
      "endPos": {
        "filename": "testdata/validation_set_attribute_lhs_not_within_write_scope/schema.keel",
        "offset": 771,
        "line": 27,
        "column": 45
      }
    },
    {
      "message": "Cannot set a field which is beyond scope of the data being created or updated",
      "hint": "Use a field which is part of a model being created or updated within this action's inputs",
      "code": "AttributeArgumentError",
      "pos": {
        "filename": "testdata/validation_set_attribute_lhs_not_within_write_scope/schema.keel",
        "offset": 876,
        "line": 30,
        "column": 18
      },
      "endPos": {
        "filename": "testdata/validation_set_attribute_lhs_not_within_write_scope/schema.keel",
        "offset": 907,
        "line": 30,
        "column": 49
      }
    },
    {
      "message": "Cannot set a field which is beyond scope of the data being created or updated",
      "hint": "Use a field which is part of a model being created or updated within this action's inputs",
      "code": "AttributeArgumentError",
      "pos": {
        "filename": "testdata/validation_set_attribute_lhs_not_within_write_scope/schema.keel",
        "offset": 999,
        "line": 33,
        "column": 18
      },
      "endPos": {
        "filename": "testdata/validation_set_attribute_lhs_not_within_write_scope/schema.keel",
        "offset": 1022,
        "line": 33,
        "column": 41
      }
    }
  ]
}